### PR TITLE
Sry klok import fix task

### PR DESCRIPTION
**Pull request by royyesud**
https://www.pivotaltracker.com/story/show/147194985

### CODE REVIEW

**Comment by amcates** [comment]
Please change tabs to 2 spaces.  Team can show you how to have Sublime do that for you.

File: lib/tasks/fix_klok_import_data.rake [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		change = prompt "Would you like to change the performer id and creator id for a specific performer id? (Y/N) "
+		if change == "Y"
+			old_performer_id = prompt "Enter performer id: "
+
+			fulfillments = Fulfillment.where("klok_entry_id IS NOT NULL AND performer_id = ?", old_performer_id)
+			no_of_fulfillments = fulfillments.count
+			puts "There are " + no_of_fulfillments.to_s + " fulfillments with performer id " + old_performer_id
+			
+			if no_of_fulfillments > 0
+				new_performer_id = prompt "Enter new performer id: "
```
Comment (amcates) [inline lib/tasks/fix_klok_import_data.rake]: You might want a way to validate the IDs are actually a number.  Also, I would write a log file of every fulfillment that you change.   That way if an error occurs we can undo the changes quickly.

**Comment by Stuart-Johnson** [comment]
@amcates So... this file IS double spaced locally, I just tested it out on mine, and committed, but github still insists that it's 4 spaces. I have no idea why.

Edit: Nevermind, figured it out, sublime was being TOO helpful

**Comment by amcates** [comment]
Looks good to me.  Now to get travis to pass.

**Review by Stuart-Johnson — APPROVED** [review]

**Review by kayla-glick — APPROVED** [review]

**Review by jleonardw9 — APPROVED** [review]

**Review by jwiel86 — APPROVED** [review]

**Review by marklohr — APPROVED** [review]
